.github/workflows/tests.yml

```diff
@@ -63,21 +75,21 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
       - name: script
+        id: test
         run: |
           forklift cargo build --locked --profile production --target x86_64-unknown-linux-musl --bin polkadot-execute-worker --bin polkadot-prepare-worker
           cd polkadot/scripts/list-syscalls
           ./list-syscalls.rb ../../../target/x86_64-unknown-linux-musl/production/polkadot-execute-worker --only-used-syscalls | diff -u execute-worker-syscalls -
           ./list-syscalls.rb ../../../target/x86_64-unknown-linux-musl/production/polkadot-prepare-worker --only-used-syscalls | diff -u prepare-worker-syscalls -
-    # todo:
-    # after_script:
-    # - if [[ "$CI_JOB_STATUS" == "failed" ]]; then
-    #   printf "The x86_64 syscalls used by the worker binaries have changed. Please review if this is expected and update polkadot/scripts/list-syscalls/*-worker-syscalls as needed.\n";
-    #   fi
+      - name: on_failure
+        if: failure() && steps.test.outcome == 'failure'
+        run: |
+          echo "The x86_64 syscalls used by the worker binaries have changed. Please review if this is expected and update polkadot/scripts/list-syscalls/*-worker-syscalls as needed." >> $GITHUB_STEP_SUMMARY
 
```

Comment:
I did it in https://github.com/paritytech/polkadot-sdk/pull/5385 so could lead to conflicts

Reply:
I checked your version and liked my implementation more 😛 but open for discussion

Reply:
then unrequire the tests, merge my, and merge conflicts here with what you like more 😅😅